README.md

```diff
@@ -4,3 +4,14 @@ github flows,
 linters, 
 github pull requests,
 and my ability to submit a project for code review
+
+# Author:
+Joshua Blue-Jack
+linkedin: Joshua Blue-Jack
+github: https://github.com/Tamunokuro
+
+# Acknowldgement: 
+AbhishekkM05
+github: https://github.com/AbhishekkM05
```

Comment:
- [ ] As requested by the previous code reviewer, please use the [readme template ](https://github.com/microverseinc/readme-template)provided by microverse for your readme. Only remove the sections that do not apply to your project such as the test section. Please add steps that one can take to get a local copy of your project under the `Getting Started` section. The steps can be;
- `clone the repo using the command git clone https://github.com/Tamunokuro/hello_microverse.git`
- `run npm install`
- etc